### PR TITLE
Display Location Data on a Chart

### PR DESCRIPTION
**Pull request by theresenieva**
Display Location Data on a Chart.

I've added buttons to the meeting page to allow users to switch between the map view and the chart view of the location data.

### CODE REVIEW

**Comment by theresenieva** [comment]
Screen Shot 2021-02-12 at 10 38 10 am

Screen Shot 2021-02-12 at 10 38 20 am

File: GoMeet/src/main/webapp/js/interactive-chart.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+google.charts.load('current', {'packages':['corechart']});
+
+/** Fetches fruit data and uses it to create a chart. */
```
Comment (anna-giang) [inline GoMeet/src/main/webapp/js/interactive-chart.js]: This comment should be updated 🍎

**Review by anna-giang — APPROVED** [review]
Looks good to me 👍 (reviewed the last 3 commits in this PR as you mentioned)

File: GoMeet/src/main/webapp/js/interactive-chart.js (hunk 1 above)
Comment (theresenieva) [reply GoMeet/src/main/webapp/js/interactive-chart.js]: Good catch! Thanks for pointing it out

**Review by ghost — APPROVED** [review]
LGTM

**Pull request merged**